### PR TITLE
Document testing live data

### LINKED ISSUE #49: Document how to test live data interface
Record in the developer documentation how to test the live data interface by launching the NeXus-Streamer and Kafka using a docker-compose script.

### PR DESCRIPTION
**Pull request by matthew-d-jones**
Closes #49

Adds a docker-compose script and documentation on how to use it to test the live data functionality.

### CODE REVIEW

File: docs/developer/testing-live-data.rst [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+          time_of_flight: [     12379936     14495801     14658190     ... truncated 756 elements ...     36832880   ]
+        }
+        ...
+
+Try using ``scippneutron.data_stream``, for example
+
+    .. code-block:: python
+
+        import asyncio
+
```
Comment (nvaytet) [inline docs/developer/testing-live-data.rst:91]: Add `import scippneutron as scn` ?

File: docs/developer/testing-live-data.rst [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+will use more and more disk space until you terminate docker-compose.
+
+Clean Up
+~~~~~~~~
+
+After you are done testing you can clean up the containers and free up used disk space by running
+
+    .. code-block:: sh
+
+        docker system prune -a --volumes
```
Comment (nvaytet) [inline docs/developer/testing-live-data.rst:112]: Does this have the potential to delete some data in other (unrelated) docker containers that the user has created from other projects?

**Review by nvaytet — CHANGES_REQUESTED** [review]
Thanks for writing this down very clearly.
Just a few comments.

File: docs/developer/testing-live-data.rst (hunk 2 above)
Comment (matthew-d-jones) [reply docs/developer/testing-live-data.rst]: I've replaced it with commands to specifically remove the live data containers and their volumes

**Review by nvaytet — APPROVED** [review]

**Pull request merged**